src/test/java/net/sf/jabref/model/entry/BibEntryTests.java

```diff
@@ -212,16 +212,21 @@ public void setNullField() {
 
     @Test(expected = NullPointerException.class)
     public void addNullKeywordThrowsNPE() {
-        keywordEntry.addKeyword(null);
+        keywordEntry.addKeyword(null, Globals.prefs.get(JabRefPreferences.KEYWORD_SEPARATOR));
```

Comment:
Right! Tests should try to never initialize `Globals.prefs`. Actually, your PR is one step in this direction since so far it was difficult to tell which preferences are accessed deep in the call chain.